spec/controllers/announcements_controller_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+# rubocop:disable Metrics/BlockLength
+RSpec.describe AnnouncementsController, type: :controller do
+  let(:episode) { create(:episode) }
+  let(:announcement) { create(:announcement, episode: episode) }
+
+  context 'with authorized user' do
+    let(:role) { create(:role) }
+    before do
+      @user = create(:user, role: role)
+
```

Comment:
All steps for user auth in controller specs better to move into spec helper

Reply:
temporary removed it
i'll add it later